src/Function/Consequences.agda

```diff
@@ -44,3 +46,12 @@ module _ (From : Setoid a ℓ₁) {≈₂ : Rel B ℓ₂} where
   inverseᵇ⇒bijective : ∀ {f f⁻¹} → Congruent ≈₂ ≈₁ f⁻¹ → Inverseᵇ ≈₁ ≈₂ f f⁻¹ → Bijective ≈₁ ≈₂ f
   inverseᵇ⇒bijective cong₂ (invˡ , invʳ) =
     (inverseʳ⇒injective cong₂ invʳ , inverseˡ⇒surjective ≈₁ ≈₂ invˡ)
+
+module _
+  {A : Set a} {B : Set b} {f : A → B}
```

Comment:
`{A : Set a} {B : Set b}` are variable so can be removed here (then the other module arguments can live on a single line).